gallery/src/pages/AdditionalResources.tsx

```diff
@@ -19,53 +19,53 @@ const AdditionalResources = () =>
       <NxTile.SubsectionHeader>
         <NxH3>Classes / Tutorials</NxH3>
       </NxTile.SubsectionHeader>
-      <ul className="nx-list nx-list--bulleted">
-        <li className="nx-list__item">
+      <NxList bulleted>
+        <NxList.Item>
           <NxTextLink external href="https://reactjs.org/tutorial/tutorial.html">
```

Comment:
Missing `NxList.Text` again.  Or do we want to change the documentation of bulleted lists to officially make `NxList.Text` optional in that case? @glenhunter

Reply:
> Missing `NxList.Text` again. Or do we want to change the documentation of bulleted lists to officially make `NxList.Text` optional in that case? @glenhunter

I think that we specified the use of `NxList.Text` to avoid bare text nodes but it's hard to see that being an issue in a bulleted list.

Reply:
> I think that we specified the use of NxList.Text to avoid bare text nodes but it's hard to see that being an issue in a bulleted list.

While true, if we're going to officially support it we need to have examples and visual tests of it.

Reply:
> > I think that we specified the use of NxList.Text to avoid bare text nodes but it's hard to see that being an issue in a bulleted list.
> 
> While true, if we're going to officially support it we need to have examples and visual tests of it.

Should be resolved here: https://github.com/sonatype/sonatype-react-shared-components/pull/491/commits/49e847b39ad47fbb73a4d350ad12678101765537